network/test/middleware_test.go

```diff
@@ -301,7 +301,7 @@ func (m *MiddlewareTestSuite) TestLargeMessageSize_SendDirect() {
 	require.NoError(m.Suite.T(), err)
 
 	// check message reception on target
-	unittest.RequireCloseBefore(m.T(), ch, 3*time.Second, "source node failed to send large message to target")
+	unittest.RequireCloseBefore(m.T(), ch, 30*time.Second, "source node failed to send large message to target")
```

Comment:
when all tests are run this test takes ~10seconds.

Reply:
Nit: 15 seconds seem more reasonable. Tests run under a global timeout of 30s by default in many contexts anyway.